.travis/Dockerfile.fedora-rawhide

```diff
@@ -0,0 +1,3 @@
+FROM fedora:rawhide
+
+RUN dnf -y install ansible dnf-plugins-core
```

Comment:
Might be worth adding a comment why *dnf-plugins-core* is necessary. I guess that's what "nothing needs to be installed on the host"-Ansible needs to install packages?

Reply:
`dnf-plugins-core` is needed for installing build dependencies  so it makes more sense to install it in the playbook. It's weird that the dnf ansible module doesn't support installing build dependencies, apt module does support it.